snap/local/src/desktop-launch

```diff
@@ -0,0 +1,356 @@
+#!/bin/bash
+#################
+# Launcher init #
+#################
+
+# shellcheck disable=SC2034
+START=$(date +%s.%N)
+
+if ! snapctl is-connected "gaming-mesa"; then
+  echo "ERROR: not connected to the gaming-mesa content interface."
+  echo "To connect:"
+  echo "sudo snap connect heroic:gaming-mesa gaming-graphics-core22"
```

Comment:
So I understand this is something every user will have to do themselves after installing the Snap? Can we do this automatically somehow?
I just envision confused users that only install Heroic, try to launch it from their application launcher, and then nothing happens as they have to run this command, which is only printed out to the (in this situation nonexistent) console

Reply:
No, this is auto-connected. This file is just for the sake of safety.